test/testRunner.js

```diff
@@ -317,35 +346,273 @@ describe('JES explorer function verification tests', () => {
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobIdFilter(driver, [expectedJobId])).to.be.true;
+                });
+            });
+
+            describe('Status Url Filter', () => {
+                it('Should handle setting status from url query');
+                it('Should handle fetching only ACTIVE jobs', async () => {
+                    const filters = { status: 'ACTIVE' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedStatus = ['ACTIVE'];
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobStatusFilter(driver, expectedStatus)).to.be.true;
+                });
+
+                // TODO:: Can't gurantee we will have jobs in INPUT state so skip until we can
+                it.skip('Should handle fetching only INPUT jobs', async () => {
+                    const filters = { status: 'INPUT' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedStatus = ['INPUT'];
+
+                    await loadUrlWithSearchFilters(driver, filters);
+
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobStatusFilter(driver, expectedStatus)).to.be.true;
+                });
+
+                it('Should handle fetching only OUTPUT jobs', async () => {
+                    const filters = { status: 'OUTPUT' };
+                    const expectedFilter = { ...DEFAULT_SEARCH_FILTERS, ...filters };
+                    const expectedStatus = ['ABEND', 'OUTPUT', 'CC', 'CANCELED', 'JCL ERROR'];
+
+                    await loadUrlWithSearchFilters(driver, filters);
+                    expect(await testFilterDisplayStringValues(driver, expectedFilter)).to.be.true;
+                    expect(await testJobStatusFilter(driver, expectedStatus)).to.be.true;
+                });
+            });
         });
     });
     describe('JES explorer spool file in url query (explorer-jes/#/viewer)', () => {
-        it('Should handle rendering expected components with viewer route (File Viewer)');
-        it('Should handle fetching file from url query string');
-        it('Should render file name, job name and job id in card header');
-        it('Should handle rendeing file contents in Orio editor');
+        const VIEWER_BASE_URL = `${BASE_URL}/#/viewer`;
+        const loadUrlWithViewerFilters = loadPageWithFilterOptions(VIEWER_BASE_URL, {}, { checkJobsLoaded: false });
+        const testFileName = 'JESJCL';
+        let testFilters;
+
+        before('get jobIds list from jobs filtered by ZOWE* prefix', async () => {
+            const filters = { prefix: 'ZOWE*' };
```

Comment:
Narrow this down to ensure we get a running ZOWE job as on my system the first job returned was an old one which had it's output files purged, 

Should do the trick:
```const filters = { prefix: ZOWE_JOB_NAME, status: 'ACTIVE' };```

Reply:
changed to above test case for this test